widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -4515,17 +4515,42 @@ div.dijitDateTextBox.dijitComboBox {
   outline-offset: var(--focus-ring-offset);
 }
 
+.fileBrowserColumn > .body {
+  display: flex;
+  flex-direction: column;
+  flex: none;
+}
+.fileBrowserColumn > .body > .folder {
+  width: max-content;
+  min-width: 100%;
+}
 .fileBrowserColumn .body .folder {
   margin-left: 30px;
   margin-top: 0;
   margin-bottom: 0;
+  display: flex;
+  flex-direction: column;
+  flex: none;
+}
+.fileBrowserColumn .body .folders {
+  display: flex;
+  flex-direction: column;
+  flex: none;
 }
 
 .fileBrowserColumn .body .folder .element {
   padding: 8px 0;
   font-size: 14px;
   font-family: opensansregular, Helvetica, Arial, Sans serif;
   position: relative;
+  display: flex;
+  flex-direction: row;
+}
+.fileBrowserColumn .body .folder .element > * {
+  flex: none;
+}
+.fileBrowserColumn .body .folder .element > .title {
+  flex: 1 0 auto;
```

Comment:
NOTE: Allowing the title to grow allows _clicking_ anywhere on the line to select, even beyond the text.